README.md

```diff
@@ -1,8 +1,107 @@
+Table of Contents
+=================
 
+* [Introduction](#introduction)
+* [Charts repository](#repository)
+  * [Available Charts in Repository](#available-charts-in-repository)
+  * [Usage](#usage)
+  * [Development](#development)
+  * [Testing](#testing)
+* [Templates](#templates)
+  * [Role-Based Access Control](#rbac)
 
-## How to play
+# Introduction
+
+This repository contains Helm templates and charts used by other applications part of the Snowdrop team.
+
+# Repository
+
+The repository has been configured to serve the static helm index and chart files. To use it:
+
+```console
+$ helm repo add snowdrop http://snowdrop.github.io/helm
+```
+
+And confirm that the snowdrop repository is listed:
+
+```console
+$ helm repo list
+NAME           	URL                               
+snowdrop	    http://snowdrop.github.io/helm
+```
+
+## Available Charts in Repository
+
+| Chart Name                     | Description | Configuration |
+|--------------------------------|-------------| --------------|
+| spring-boot-example-app        | Chart to be used by the Snowdrop Spring Boot examples | [values.yaml](repository/spring-boot-example-app/values.yaml) |
+
+## Usage
+
+To use one of the available charts, for example `spring-boot-example-template`, update or add the `requirements.yaml` file to register the dependency:
+
+```yaml
+dependencies:
+  - name: spring-boot-example-app
+    version: 0.0.1
+    repository: http://snowdrop.github.io/helm
+```
+
+Now, download the dependencies using: 
+```console
+$ helm dependency update
+```
+
+Configure the template accordingly using the chart name as root element:
+
+```yaml
+spring-boot-example-app: # This is the chart name from the dependencies!
+  name: rest-http
+  version: 2.5.0-0-SNAPSHOT
+  s2i:
+    source:
+      repo: https://github.com/snowdrop/rest-http-example
+      ref: sb-2.5.x
+```
+
+And, install your helm:
+```console
+$ helm install rest-http .
+```
+
+## Development
+
+To add a new chart in the repository, you need to follow the following steps:
+1. Add the new chart under `repository/<new chart name>`
+2. Update the [Makefile](Makefile) file to add the new repository name in the `CHARTS` array
+3. Run `make release` from the root repository folder
+
+## Testing
+
+1. Expose the repository locally using Docker and chartmuseum:
```

Comment:
What is chartmuseum and why do we have to use it ? Maybe add a link to the prohect chartmuseum ?

Reply:
Changed. Let me know if it looks clearer now.